db/postgres/mod_links.go

```diff
@@ -0,0 +1,121 @@
+package postgres
```

Comment:
Please rename the file to `mod_archs.go` and all the appropriate names as well.

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/bdd565d8997568955b8e33bcb352c38c7af5c2c5